### PR TITLE
dev/reference: add some description about max_execution_time

### PR DESCRIPTION
**Pull request by tiancaiamao**
### What is changed, added or deleted? 

Tiny update the document.

### What is the related PR or file link(s)? 

https://github.com/pingcap/docs/pull/1430

### Which version does your change affect? 

dev, v3.0, v2.1

### Checklist 

- [x] Add a new file to `TOC.md`
- [x] No Tab spaces anywhere 
- [x] Leave a blank line both before and after a code block
- [x] Keep the first level heading consistent with `title` in metadata
- [x] Use *four* spaces for each level of indentation except that in `TOC.md`

### CODE REVIEW

**Comment by tiancaiamao** [comment]
PTAL @jackysp @dcalvin

**Comment by tiancaiamao** [comment]
Done @jackysp

**Comment by lilin90** [comment]
@jackysp PTAL again~ Thanks!

File: dev/reference/configuration/tidb-server/mysql-variables.md [hunk 1]
```diff
@@ -106,6 +106,9 @@ MySQL 系统变量 (System Variables) 是一些系统参数，用于调整数据
 | sql_mode | GLOBAL \| SESSION | 支持部分 MySQL SQL mode，|
 | time_zone | GLOBAL \| SESSION | 数据库所使用的时区 |
 | tx_isolation | GLOBAL \| SESSION | 事务隔离级别 |
+| max\_execution\_time | GLOBAL \| SESSION | 语句超时时间，单位为毫秒 |
+
+注意，`max_execution_time` 目前对所有类型的 `statement` 生效，并非只对 `select` 语句生效。实际精度在 100ms 级别，而不是准确的毫秒级别。
```
Comment (lilin90) [inline dev/reference/configuration/tidb-server/mysql-variables.md]:
```suggestion
> **注意：**
>
> `max_execution_time` 目前对所有类型的 `statement` 生效，并非只对 `SELECT` 语句生效。实际精度在 100ms 级别，而非更准确的毫秒级别。
```

File: v2.1/reference/configuration/tidb-server/mysql-variables.md [hunk 2]
```diff
@@ -106,6 +106,9 @@ MySQL 系统变量 (System Variables) 是一些系统参数，用于调整数据
 | sql_mode | GLOBAL \| SESSION | 支持部分 MySQL SQL mode，|
 | time_zone | GLOBAL \| SESSION | 数据库所使用的时区 |
 | tx_isolation | GLOBAL \| SESSION | 事务隔离级别 |
+| max\_execution\_time | GLOBAL \| SESSION | 语句超时时间，单位为毫秒 |
+
+ 注意，`max_execution_time` 目前对所有类型的 `statement` 生效，并非只对 `select` 语句生效。实际精度在 100ms 级别，而不是准确的毫秒级别。
```
Comment (lilin90) [inline v2.1/reference/configuration/tidb-server/mysql-variables.md]:
```suggestion
> **注意：**
>
> `max_execution_time` 目前对所有类型的 `statement` 生效，并非只对 `SELECT` 语句生效。实际精度在 100ms 级别，而非更准确的毫秒级别。
```

File: dev/reference/performance/optimizer-hints.md [hunk 3]
```diff
@@ -37,3 +37,15 @@ SELECT /*+ TIDB_HJ(t1, t2) */ * from t1，t2 where t1.id = t2.id
 ```
 
 提示优化器使用 Hash Join 算法，这个算法多线程并发执行，执行速度较快，但会消耗较多内存。
+
+### MAX\_EXECUTION\_TIME(N)
+
+在 SELECT 语句中可以使用 `MAX\_EXECUTION\_TIME(N)`，它会限制语句的执行时间不能超过 `N` 毫秒，否则服务器会终止这条语句的执行。
```
Comment (lilin90) [inline dev/reference/performance/optimizer-hints.md]:
```suggestion
在 `SELECT` 语句中可以使用 `MAX_EXECUTION_TIME(N)`，它会限制语句的执行时间不能超过 `N` 毫秒，否则服务器会终止这条语句的执行。
```

File: v2.1/reference/performance/optimizer-hints.md [hunk 4]
```diff
@@ -37,3 +37,15 @@ SELECT /*+ TIDB_HJ(t1, t2) */ * from t1，t2 where t1.id = t2.id
 ```
 
 提示优化器使用 Hash Join 算法，这个算法多线程并发执行，执行速度较快，但会消耗较多内存。
+
+### MAX\_EXECUTION\_TIME(N)
+
+ 在 SELECT 语句中可以使用 `MAX\_EXECUTION\_TIME(N)`，它会限制语句的执行时间不能超过 `N` 毫秒，否则服务器会终止这条语句的执行。
```
Comment (lilin90) [inline v2.1/reference/performance/optimizer-hints.md]:
```suggestion
 在 `SELECT` 语句中可以使用 `MAX_EXECUTION_TIME(N)`，它会限制语句的执行时间不能超过 `N` 毫秒，否则服务器会终止这条语句的执行。
```

File: v2.1/reference/performance/optimizer-hints.md [hunk 5]
```diff
@@ -37,3 +37,15 @@ SELECT /*+ TIDB_HJ(t1, t2) */ * from t1，t2 where t1.id = t2.id
...
+
+ 例如，下面例子设置了 1 秒超时：
+
+ ```sql
+SELECT /*+ MAX_EXECUTION_TIME(1000) */ * FROM t1 INNER JOIN t2 WHERE ...
+```
+
+ 除了 hint 之外，环境变量 `max_execution_time`，也会对语句执行时间进行限制。
```
Comment (lilin90) [inline v2.1/reference/performance/optimizer-hints.md]:
```suggestion
 除了 hint 之外，环境变量 `max_execution_time` 也会对语句执行时间进行限制。
```

File: v3.0/reference/configuration/tidb-server/mysql-variables.md [hunk 6]
```diff
@@ -108,6 +108,9 @@ MySQL 系统变量 (System Variables) 是一些系统参数，用于调整数据
 | sql_mode | GLOBAL \| SESSION | 支持部分 MySQL SQL mode，|
 | time_zone | GLOBAL \| SESSION | 数据库所使用的时区 |
 | tx_isolation | GLOBAL \| SESSION | 事务隔离级别 |
+| max\_execution\_time | GLOBAL \| SESSION | 语句超时时间，单位为毫秒 |
+
+ 注意，`max_execution_time` 目前对所有类型的 `statement` 生效，并非只对 `select` 语句生效。实际精度在 100ms 级别，而不是准确的毫秒级别。
```
Comment (lilin90) [inline v3.0/reference/configuration/tidb-server/mysql-variables.md]:
```suggestion
> **注意：**
>
> `max_execution_time` 目前对所有类型的 `statement` 生效，并非只对 `SELECT` 语句生效。实际精度在 100ms 级别，而非更准确的毫秒级别。
```

File: v3.0/reference/performance/optimizer-hints.md [hunk 7]
```diff
@@ -38,3 +38,15 @@ SELECT /*+ TIDB_HJ(t1, t2) */ * from t1，t2 where t1.id = t2.id
 ```
 
 提示优化器使用 Hash Join 算法，这个算法多线程并发执行，执行速度较快，但会消耗较多内存。
+
+### MAX\_EXECUTION\_TIME(N)
+
+ 在 SELECT 语句中可以使用 `MAX\_EXECUTION\_TIME(N)`，它会限制语句的执行时间不能超过 `N` 毫秒，否则服务器会终止这条语句的执行。
```
Comment (lilin90) [inline v3.0/reference/performance/optimizer-hints.md]:
```suggestion
在 `SELECT` 语句中可以使用 `MAX_EXECUTION_TIME(N)`，它会限制语句的执行时间不能超过 `N` 毫秒，否则服务器会终止这条语句的执行。
```

File: v3.0/reference/performance/optimizer-hints.md [hunk 8]
```diff
@@ -38,3 +38,15 @@ SELECT /*+ TIDB_HJ(t1, t2) */ * from t1，t2 where t1.id = t2.id
...
+
+ 例如，下面例子设置了 1 秒超时：
```
Comment (lilin90) [inline v3.0/reference/performance/optimizer-hints.md]:
```suggestion
例如，下面例子设置了 1 秒超时：
```

File: v3.0/reference/performance/optimizer-hints.md [hunk 9]
```diff
@@ -38,3 +38,15 @@ SELECT /*+ TIDB_HJ(t1, t2) */ * from t1，t2 where t1.id = t2.id
...
+
+ ```sql
```
Comment (lilin90) [inline v3.0/reference/performance/optimizer-hints.md]:
```suggestion
```sql
```

File: v3.0/reference/performance/optimizer-hints.md [hunk 10]
```diff
@@ -38,3 +38,15 @@ SELECT /*+ TIDB_HJ(t1, t2) */ * from t1，t2 where t1.id = t2.id
...
+SELECT /*+ MAX_EXECUTION_TIME(1000) */ * FROM t1 INNER JOIN t2 WHERE ...
+```
+
+ 除了 hint 之外，环境变量 `max_execution_time`，也会对语句执行时间进行限制。
```
Comment (lilin90) [inline v3.0/reference/performance/optimizer-hints.md]:
```suggestion
 除了 hint 之外，环境变量 `max_execution_time` 也会对语句执行时间进行限制。
```

File: dev/reference/performance/optimizer-hints.md [hunk 11]
```diff
@@ -37,3 +37,15 @@ SELECT /*+ TIDB_HJ(t1, t2) */ * from t1，t2 where t1.id = t2.id
...
+
+例如，下面例子设置了 1 秒超时：
+
+```sql
+SELECT /*+ MAX_EXECUTION_TIME(1000) */ * FROM t1 INNER JOIN t2 WHERE ...
+```
+
+除了 hint 之外，环境变量 `max_execution_time`，也会对语句执行时间进行限制。
```
Comment (lilin90) [inline dev/reference/performance/optimizer-hints.md]:
```suggestion
除了 hint 之外，环境变量 `max_execution_time` 也会对语句执行时间进行限制。
```

File: v2.1/reference/performance/optimizer-hints.md [hunk 12]
```diff
@@ -37,3 +37,15 @@ SELECT /*+ TIDB_HJ(t1, t2) */ * from t1，t2 where t1.id = t2.id
 ```
 
 提示优化器使用 Hash Join 算法，这个算法多线程并发执行，执行速度较快，但会消耗较多内存。
+
+### MAX\_EXECUTION\_TIME(N)
+
+ 在 SELECT 语句中可以使用 `MAX\_EXECUTION\_TIME(N)`，它会限制语句的执行时间不能超过 `N` 毫秒，否则服务器会终止这条语句的执行。
+
+ 例如，下面例子设置了 1 秒超时：
```
Comment (lilin90) [inline v2.1/reference/performance/optimizer-hints.md]:
```suggestion
例如，下面例子设置了 1 秒超时：
```

File: v2.1/reference/performance/optimizer-hints.md [hunk 13]
```diff
@@ -37,3 +37,15 @@ SELECT /*+ TIDB_HJ(t1, t2) */ * from t1，t2 where t1.id = t2.id
...
+
+ ```sql
```
Comment (lilin90) [inline v2.1/reference/performance/optimizer-hints.md]:
```suggestion
```sql
```

**Comment by lilin90** [comment]
@jackysp PTAL

**Comment by jackysp** [comment]
LGTM

**Pull request merged**